src/passes/rejectUnsupportedFeatures.ts

```diff
@@ -87,10 +86,6 @@ export class RejectUnsupportedFeatures extends ASTMapper {
     this.visitExpression(node, ast);
   }
 
-  visitInlineAssembly(node: InlineAssembly, _ast: AST): void {
-    this.addUnsupported('Yul blocks are not supported', node);
```

Comment:
Could we check for YUL operators searching for mem and storage?

Reply:
With YulNodes implemented we would probably move the whole verifier here. I don't think it's a good idea to reject memory and storage functions here and have verification partitioned between unsupported features pass and the verifier. The idea of the verifier was to print all errors regarding the inline assembly block at once